manuscript/manuscript.md

```diff
@@ -70,6 +70,14 @@ organization:
 
 # Introduction {-}
 
+Data is the central currency of science.  However the nature of scientific data has changed dramatically with the rapid pace of technology, leading to an increasing heterogeneity. Improvements in high throughput DNA sequencing and sustained institutional support for large sensor networks [@Reid2014, @Hampton2013] have lead to the creation of massive quanities of data. At the same time collaboration between researchers is becoming increasingly common [@Adams2012] with increased coordination between researchers collecting data [@Fraser2013]. These changes mean that data is more heterogenous than ever before in both size and complexity. It ranges from petabytes of information stored in hadoop clusters to Excel spreadsheets and lab notebooks on shelves.  
+
+
+While much has been written about both the virtues of data sharing [@Wolkovich2012, @Roche2014] and best practices to do so [@White2013, @Goodman2014] how to store data is a much less discussed topic. Proper storage of data is a prerequisite for any sharing, and indeed lack of proper storage may also contribute to the phenomenon of data decay, as time passes data is less and less accessible (publicly shared or not) [ @Pepe2014, @Vines2014]. Best practices for data storage often begin and end with "use a community standard repository" and this is by all means a great practice. However data storage policies are highly variable between repositories [@Marcial2010] and data best storage practices will facilitate transition from local storage to repository. Furthermore your data may not fit with an existing repository, or only derived data products (vs raw data) are suitable, or an existing repository may have lax standards. What follows are 10 simple rules for data storage. These ideas grew out of a long discussion between Software Carpentry instructors, scientists from diverse backgrounds who have encountered a variety of data storage challenges.
```

Comment:
I added a citation to this, but I think it might be awkward to have an explanation here.